.pre-commit-config.yaml

```diff
@@ -29,14 +29,14 @@ repos:
       - id: gitleaks
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

Comment:
Don't think we want to update to an alpha version

Reply:
😦 , that sounds right

Reply:
I've removed that change